packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -294,15 +297,18 @@ export default function NavigationSubmenuEdit( {
 	const itemLabelPlaceholder = __( 'Add text…' );
 	const ref = useRef();
 
+	const [ , { canCreate: userCanCreatePages } ] =
+		useResourcePermissions( 'pages' );
```

Comment:
This part isn't the most intuitive with unused first param. What was the reason `hasResolved` wasn't merged with all other return values?

The code example included in the package presents the example that fits better here:

https://github.com/WordPress/gutenberg/blob/2026f13a17381f5bf36c84b141db071ccb27a028/packages/core-data/src/hooks/use-resource-permissions.ts#L50

Was it a sign? 😄

Reply:
I agree, I just updated the function signature